src/tests/multihost/ad/test_adparameters.py

```diff
@@ -756,3 +756,56 @@ def test_0018_bz1734040(multihost, adjoin, create_aduser_group):
         client.restore_sssd_conf()
 
         assert find.search(log), "Expected log record is missing."
+
+    @pytest.mark.tier1_3
+    def test_0019_bz2070189(self, multihost, adjoin):
```

Comment:
if the test is not using the class, it can be marked as @staticmethod and self can be removed.
https://www.digitalocean.com/community/tutorials/python-static-method